### ISSUE TITLE
Screenshots are dead links

### ISSUE THREAD

**Issue by gausie** [stephenharris/wp-readme-to-markdown#1]
Example: https://github.com/gausie/menu-image/tree/feature-image-sizes-filter

I don't know why, but it seems to be that the images do not show properly.

**Comment by MickeyKay** [comment]
Update on this one. The URL you're shooting for doesn't seem to work for me, instead it looks like the pattern I'm getting is:

`https://ps.w.org/{plugin-name}/assets/screenshot-1.png`

**Comment by zviryatko** [comment]
This PR will fix url https://github.com/stephenharris/wp-readme-to-markdown/pull/6 and for fix images need to set svn mime-type via this bash commands:
```
$ svn propset svn:mime-type image/png screenshot-1.png
$ svn commit
```

**Comment by stephenharris** [comment]
Fixed with #6. The few plug-ins I've checked are now using the new URL structure. This is a moving target so I can't guarantee the update won't break things for existing user, but in any case, this is just the default value of a configurable option. 

Will publish shorty.
